ext/compressors/iaa/iaacodec/src/IAACompressionCodecDeflate.cpp

```diff
@@ -58,13 +59,25 @@ DeflateJobHWPool::~DeflateJobHWPool()
 HardwareCodecDeflate::HardwareCodecDeflate(WT_COMPRESSOR *compressor, WT_SESSION *session)
 {
     hwEnabled = DeflateJobHWPool::instance(compressor, session).jobPoolReady();
+    pageSize = (int)getpagesize();
 }
 
 HardwareCodecDeflate::~HardwareCodecDeflate()
 {
     // Nothing to do.
 }
 
+inline void
+HardwareCodecDeflate::memPageSet(uint8_t *dest, int len) const
```

Comment:
This is always called with `len` converted from a `uint32_t`. I think it'd be cleaner to have this take a `uint32_t` (and drop the `len < 0` check).

Reply:
I agree with Will

Reply:
Already update, please review, Thanks!